File: src/library/scala/runtime/java8/JFunction0$mcJ$sp.java

/*
 * Copyright (C) 2012-2015 Typesafe Inc. <http://www.typesafe.com>
 */

package scala.runtime.java8;

@FunctionalInterface
public interface JFunction0$mcJ$sp extends scala.Function0, java.io.Serializable {
    long apply$mcJ$sp();

    default Object apply() { return scala.runtime.BoxesRunTime.boxToLong(apply$mcJ$sp()); }
}

File: src/library/scala/runtime/java8/JFunction0$mcJ$sp.scala
/*
 * Copyright (C) 2012-2015 Typesafe Inc. <http://www.typesafe.com>
 */
package scala.runtime.java8

@FunctionalInterface trait JFunction0$mcJ$sp extends Function0[Any] with Serializable {
  def apply$mcJ$sp: Long
  override def apply: Any = scala.runtime.BoxesRunTime.boxToLong(apply$mcJ$sp)
}